atecc608a_se.c

```diff
@@ -372,8 +363,7 @@ psa_status_t atecc608a_write(uint16_t slot, size_t offset, const uint8_t *data,
     psa_status_t status = PSA_ERROR_GENERIC_ERROR;
 
     /* The hardware has slots 0-15 */
-    if (slot > 15)
-    {
+    if (slot > 15) {
```

Comment:
This function doesn't go through the core, since it does not have such API (yet?). It's a convenience function, at least for now. `atcab_write_bytes_zone` returns `ATCA_BAD_PARAM` if the zone is `ATCA_ZONE_DATA` and slot is over 15, so we could skip this error check.

Reply:
Better for code size to remove the redundant check, then